src/views/v2/commerce/checkout/header/title.php

```diff
@@ -26,6 +26,11 @@
  * @var int              $gateways_active       [Global] The number of active gateways.
  */
 
+
+if ( empty( $items ) ) {
```

Comment:
We shouldn't be removing this one if the cart is empty. It was part of the designs.